src/main/scala/cc/factorie/app/nlp/load/XMLSectionalizer.scala

```diff
@@ -70,32 +72,44 @@ class XMLSectionalizer(boundaryToken:String, excludeTokens:Set[String]) extends
           }
           stateStack.pop()
           tokenBuffer += t
+//          println("token:"+t + "\t" + "accepted close tag" + "\tUsable" )
         case (excludedOpenTag(tag), Usable) =>
           if(tokenBuffer.nonEmpty) {
-            sectionBuffer += new UsableText(tokenBuffer)
+            // NOTE (Ajay, Melisa - 5/3/2016) : To filter out the following tags in text <P> , </P>, =(FOTOS)=
+            sectionBuffer += new UsableText(tokenBuffer.filter(x => !(x.string.contentEquals("<P>") ||

```

Comment:
We haven't seen changes including or not including 'p' in the excludeTokens or probably we tested wrong. So If we have this document:
<text>
<p>
This is a text
</p>
</text>
and use 'p' as an exludeTokens, it should ignore the tag 'p' AND its content or just the tag name?